internal/test-utils/src/androidMain/kotlin/coil3/test/utils/bitmaps.android.kt

```diff
@@ -100,7 +129,7 @@ fun Bitmap.isSimilarTo(
  */
 fun Bitmap.assertIsSimilarTo(
     expected: Bitmap,
-    threshold: Double = 0.99
+    threshold: Double = 0.987
```

Comment:
Let's use `0.98`

Reply:
I changed it back to 0.99 in android and 0.98 in common.